Code/Mantid/Framework/Algorithms/test/ConvertToMatrixWorkspaceTest.h

```diff
@@ -39,19 +39,23 @@ class ConvertToMatrixWorkspaceTest : public CxxTest::TestSuite
 
   void testExec_2D_to_2D()
   {
+	Mantid::Algorithms::ConvertToMatrixWorkspace cloner;
     if ( !cloner.isInitialized() ) cloner.initialize();
-
-    Mantid::DataHandling::LoadRaw3 loader;
-    loader.initialize();
-    loader.setPropertyValue("Filename", "LOQ48127.raw");
-    loader.setPropertyValue("OutputWorkspace", "in");
-    loader.execute();
-
+	//create 2D input workspace
+	Mantid::API::MatrixWorkspace_sptr in = WorkspaceCreationHelper::Create2DWorkspace(5,10);
+	//add instance to variable 'in' 
+	Mantid::API::AnalysisDataService::Instance().addOrReplace("in", in);
```

Comment:
Better not to use the AnalysisDataService at all. When you create the algorithm, call Algorithm::setChild(true), then you can just call Algorithm::getProperty("OutputWorkspace") after execute() to fetch the workspace shared pointer directly. Ask me or Martyn about this.